python/lsst/dax/ppdb/sql/_ppdb_sql.py

```diff
@@ -131,6 +131,12 @@ def __init__(self, config: PpdbConfig):
         # Check schema version compatibility
         self._versionCheck(self._metadata, schema_version)
 
+        # Check if schema uses MJD TAI for timestamps (DM-52215).
+        self._use_mjd_tai = False
+        for table in self._sa_metadata.tables.values():
+            if table.name == "DiaObject":
+                self._use_mjd_tai = "validityStartMjdTai" in table.columns
```

Comment:
This would be better if the information came from an APDB interface rather than doing this manual check on a hard-coded fieldname here.